src/main/java/com/semmle/jira/addon/workflow/LgtmDismissAlert.java

```diff
@@ -32,6 +32,7 @@
  */
 public class LgtmDismissAlert extends AbstractJiraFunctionProvider {
   public static final String FIELD_TRANSITION = "transitionField";
+  static final String FULL_MODULE_KEY = "com.semmle.lgtm-jira-addonlgtm-dismiss-alert";
```

Comment:
Was this supposed to have another `.` in it?

Reply:
No, I don't think so.

Jira seems to simply join the plugin ID and the function ID. See for example:
```
<arg name="full.module.key">
com.atlassian.jira.plugin.system.workflowupdate-issue-field-function
</arg>
```